as-controller-board-status/functions/models.go

```diff
@@ -64,10 +67,16 @@ type CheckBoardStatus struct {
 	DoorClosed              bool              // true means the door is closed and false means the door is open
 	Measurements            []TempMeasurement // used to store temperature readings over time.
 	LastNotified            time.Time         // used to store last time a notification was sent out so we don't spam the maintenance person
-	Configuration           *ControllerBoardStatusAppSettings
+	Configuration           *config.ControllerBoardStatusConfig
 	SubscriptionClient      interfaces.SubscriptionClient
 	NotificationClient      interfaces.NotificationClient
 	ControllerBoardStatus   *ControllerBoardStatus
+	averageTemperatureMeasurement             time.Duration 
+	notificationSubscriptionRESTRetryInterval time.Duration 
```

Comment:
some variables are in lower case and some are in Capitalized first letter, should we make them consistent?

Reply:
Only is they are all meant to be public, otherwise it is ok to have some private.

Reply:
I left them lower case to make them private, this way they can only be changed via configuration.toml

Reply:
make sense!